web-client/terraform/bin/deploy-app.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+ENVIRONMENT=$1
+
+BUCKET=gov.ustaxcourt.ef-cms.terraform.deploys
```

Comment:
(Placeholder to indicate that the same question I asked about this variable's value in `document-service` applies here, too.)

Reply:
This is actually ok (versus the hard coded flexion domain was bad above).  All the terraform related code is currently stored in a single bucket.